test/integration/targets/fortios_ipv4_policy/tasks/test_params.yml

```diff
@@ -0,0 +1,74 @@
+---
+ - name: Forget id
+   fortios_ipv4_policy:
+     file_mode: true
+     config_file: "{{role_path}}/files/default_config.conf"
+     # id: 42
+     src_addr: all
+     dst_addr: all
+     policy_action: accept
+     service: ALL
+     state: present
+   register: forget_id
+   ignore_errors: True
+
+ - name: Forget src_addr
+   fortios_ipv4_policy:
+     file_mode: true
+     config_file: "{{role_path}}/files/default_config.conf"
+     id: 42
+     # src_addr: all
+     dst_addr: all
+     policy_action: accept
+     service: ALL
+     state: present
+   register: forget_src_addr
+   ignore_errors: True
+
+ - name: Forget dst_addr
+   fortios_ipv4_policy:
+     file_mode: true
+     config_file: "{{role_path}}/files/default_config.conf"
+     id: 42
+     src_addr: all
+     # dst_addr: all
+     policy_action: accept
+     service: ALL
+     state: present
+   register: forget_dst_addr
+   ignore_errors: True
+
+ - name: Forget policy_action
+   fortios_ipv4_policy:
+     file_mode: true
+     config_file: "{{role_path}}/files/default_config.conf"
+     id: 42
+     src_addr: all
+     dst_addr: all
+     # policy_action: accept
```

Comment:
It's not left over.
I assert that missing required param thow an exception, for all these test, I comment the param line.